docs/features/bgp-integration/route-advertisements.md

```
@@ -0,0 +1,629 @@
+for the selected networks to instruct the OVN and host network controllers on
+each node to reconfigure the network.
+
+#### FRRConfiguration instances generated by OVN-Kubernetes
+
+When `RouteAdvertisements` instances are configured, OVN-Kubernetes generates
+additional `FRRConfiguration` instances in order for the selected network
+prefixes to be advertised, using the following logic:
+
+- For each pair combination of selected network and selected node; and for each
+  selected `FRRConfiguration` to be used as template:
+    - If the `FRRConfiguration` does not apply to the node, it is discarded.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)